swarm.go

```diff
@@ -241,6 +244,10 @@ func NewSwarm(config *api.Config, mockStore *mock.NodeStore) (self *Swarm, err e
 		pss.SetHandshakeController(self.ps, pss.NewHandshakeParams())
 	}
 
+	pubsub := pss.NewPubSub(self.ps)
+	self.pushSync = pushsync.NewPusher(localStore, pubsub, self.tags)
```

Comment:
Since push sync is constructed here, it should be closed in the Swarm.Stop before netstore close.